src/store/initial_state.js

```diff
@@ -511,14 +511,6 @@ const state: GlobalState = {
                 status: 'not_started',
```

Comment:
All the roles objects(getRolesByNames, getRoleByName, getRole and editRole) can be removed here.